cluster/examples/kubernetes/ceph/operator.yaml

```diff
@@ -204,7 +204,7 @@ rules:
   - update
   - delete
 - apiGroups:
-  - extensions
+  - apps
```

Comment:
if there are changes to the rbac i would have expected the upgrade_test.go to need an update, otherwise the rbac will not be updated in the test. See [here](https://github.com/rook/rook/blob/master/tests/integration/upgrade_test.go#L154) for an example of updating the rbac for a similar change. The upgrade test is what helps us put the final upgrade documentation ready when it's time for release.

Reply:
added